### DISCUSSION TITLE
Apply rounded corner NativeControlHost

### DISCUSSION THREAD

**Discussion by XDgl233** [AvaloniaUI/Avalonia#13978] [Q&A]
Hello! I am working on a project using Libvlcsharp, so far I find it struggling to create rounded corner nativecontrolhost (in windows). Is it something related to dwmapi.dll? Thanks in advance :)

**Comment by maxkatz6** [comment]
No, it's not a rendered control, and most of Avalonia properties can't be applied there. This control is also rendered on top of Avalonia content, as a native control.
